src/typeahead/typeahead.js

```diff
@@ -162,6 +162,7 @@ angular.module('ui.bootstrap.typeahead', ['ui.bootstrap.debounce', 'ui.bootstrap
 
     //pop-up element used to display matches
     var popUpEl = angular.element('<div uib-typeahead-popup></div>');
+    var clearBtnEl = angular.element('<span class="glyphicon glyphicon-remove" ng-style="{ cursor:\'pointer\',position:\'absolute\',top:positionClearBtn.top+\'px\', left:positionClearBtn.left+\'px\'}" ng-show="showClearBtn" ng-click="clearModel()"></span>');
```

Comment:
It would be nicer to change this to something similar to how the popUpEl attributes are added (just the line below), i.e.
```js
var clearBtnEl = angular.element('<span></span>');
clearBtnEl.attr({
  class: 'glyphicon glyphicon-remove',
  style: 'position: absolute; cursor: pointer;',
  ngStyle: '...',
  ...
});
```